### ISSUE TITLE
Fully support Python 3

### ISSUE THREAD

**Issue by elyscape** [cloudflare/python-cloudflare#47]
This module is not fully compatible with Python 3. There have been various attempts to fix this (c.f. #1, #3, #14, #21), but none of them have been merged and they were all incomplete. One particular area that hasn't been addressed in any of the pull requests is `/examples`, which relies heavily on Python 2 behavior (primarily `print` as a statement rather than a function). Please finish the migration to support Python 3. This is holding up packaging of the module for Fedora, which in turn is holding up packaging of [the certbot-dns-cloudflare module] for Fedora/EPEL.

[the certbot-dns-cloudflare module]: https://pypi.python.org/pypi/certbot-dns-cloudflare

**Comment by mahtin** [comment]
Hello @elyscape - yes, I've been quite negligent with the python3 porting work; mainly because of some non-python3 issues with the libraries. However, while trying to squash a unicode bug, I believe I've also done all the work needed to make this library fully python2/python3 compatible. I've just pushed the **2.0.4** release over to [GitHub](https://github.com/cloudflare/python-cloudflare) and [PyPI](https://pypi.python.org/pypi/cloudflare).

I've also fixed ``/examples``.

Let me know if this works for you.
